internal/system/agent/executor/os.go

```diff
@@ -28,7 +28,8 @@ func findAndStopProcess(output string, err error, process string) error {
 
 	var pid int
 	if err != nil {
-		log.Fatalf("Error during findAndStopProcess() as follows: " + err.Error())
+		log.Printf("Error during findAndStopProcess() as follows: %s" + err.Error())
```

Comment:
This line should be

`log.Printf("Error during findAndStopProcess() as follows: %s" , err.Error())`